pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1441,6 +1441,7 @@ func doInactivate(ctx *domainContext, status *types.DomainStatus, impatient bool
 			status.DomainId, status.BootTime.Format(time.RFC3339Nano),
 			status.Key())
 	} else if err != nil {
+		domainID = status.DomainId
```

Comment:
All the existing code uses status.DomainId, so if you do that in your new call you don't need this; can keep domainID as a "local" thing here at the top.
In fact you can make that more obvious by changing line 1436 1437 to 
if domainID, _, err := ... ; err == nil && domainID != status.DomainId {

Reply:
I removed domainIDs from tasks, seems they are not needed